### PR TITLE
YWOE stormkeld_curator_giant_secrets.txt + support

### PR DESCRIPTION
**Pull request by Northmoc**
script credit @Simisays (note there were a couple extra lines on the Curator I removed)

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/stormkeld_curator_giant_secrets.txt [hunk 1]
```diff
@@ -0,0 +1,22 @@
+Name:Stormkeld Curator
+ManaCost:4 W
+Types:Creature Giant
+PT:6/6
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigAurify | TriggerDescription$ When CARDNAME enters the battlefield, you may put any number of Aura cards from your graveyard and/or hand onto the battlefield attached to it.
+SVar:TrigAurify:DB$ ChangeZone | Origin$ Hand | Destination$ Battlefield | ChangeType$ Aura.CanEnchantSource+YouOwn | AttachedTo$ Self | ChangeNum$ HandX | Optional$ True | Hidden$ True | SubAbility$ GraveAuras
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/stormkeld_curator_giant_secrets.txt:6]: any reason it can't be combined into a single ChangeZone?

Comment (Hanmac) [reply forge-gui/res/cardsfolder/upcoming/stormkeld_curator_giant_secrets.txt:6]: the Auras probably needs to enter at the same time

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/stormkeld_curator_giant_secrets.txt:6]: Yeah I copied this part almost entirely from Bruna, Light of Alabaster (mistakenly) thinking they were the same rules wise 😱

**Review by tool4ever — APPROVED** [review]

File: forge-gui/res/cardsfolder/b/bruna_light_of_alabaster.txt [hunk 2]
```diff
@@ -6,12 +6,12 @@ K:Flying
...
-SVar:Aurify:DB$ RepeatEach | RepeatSubAbility$ BrunaAttach | RepeatCards$ Aura.CanEnchantSource+NotAttachedTo | SubAbility$ HandAuras
+SVar:Aurify:DB$ RepeatEach | RepeatSubAbility$ BrunaAttach | RepeatCards$ Aura.CanEnchantSource+NotAttachedTo | SubAbility$ ZoneAuras
 SVar:BrunaAttach:DB$ Attach | Object$ Remembered | Defined$ Self | Optional$ True
-SVar:HandAuras:DB$ ChangeZone | Origin$ Hand | Destination$ Battlefield | ChangeType$ Aura.CanEnchantSource+YouCtrl | AttachedTo$ Self | ChangeNum$ HandX | Optional$ True | Hidden$ True | SubAbility$ GraveAuras
-SVar:GraveAuras:DB$ ChangeZone | Origin$ Graveyard | Destination$ Battlefield | ChangeType$ Aura.CanEnchantSource+YouCtrl | AttachedTo$ Self | ChangeNum$ GraveX | Optional$ True | Hidden$ True
-SVar:HandX:Count$ValidHand Aura.CanEnchantSource+YouCtrl
-SVar:GraveX:Count$ValidGraveyard Aura.CanEnchantSource+YouCtrl
+SVar:ZoneAuras:DB$ ChangeZone | Origin$ Hand,Graveyard | Destination$ Battlefield | ChangeType$ Aura.CanEnchantSource+YouOwn | AttachedTo$ Self | ChangeNum$ Count | Optional$ True | Hidden$ True
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/b/bruna_light_of_alabaster.txt:11]: in the future we should add some special values like "Any" or "All" so we don't need the useless Count below

**Pull request merged**